libs/SalesforceSDKCommon/SalesforceSDKCommonTests/KeychainItemManagerTests.swift

```diff
@@ -26,29 +26,17 @@
 //  WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import XCTest
-import SalesforceSDKCommon
-enum KeychainTestError: Error {
-    case failed(osStatus: OSStatus)
-    case failedToRead
-}
-
-class KeychainItemManagerTests: XCTestCase {
+@testable import SalesforceSDKCommon
```

Comment:
Did something need internal access?

Reply:
You are right, not needed since we are only testing public methods.